src/main/kotlin/io/easybreezy/project/application/issue/queryobject/Issue.kt

```diff
@@ -25,15 +32,20 @@ class HasIssuesInCategoryQO(private val inCategory: UUID) : QueryObject<Boolean>
 
 class HasIssuesInStatusQO(private val inStatus: UUID) : QueryObject<Boolean> {
     override suspend fun getData() =
-        Issues.select { Issues.status eq inStatus }.count() > 0
+        Workflows.select { Workflows.status eq inStatus }.count() > 0
 }
 
 class IssueQO(private val id: UUID) : QueryObject<IssueDetails> {
     override suspend fun getData() =
         Issues
             .leftJoin(IssueLabel)
+            .leftJoin(Comments)
```

Comment:
Do you need comments on the issue view?
I think we can add our own endpoints for  comments, participants

Reply:
ok